docs/core-concepts/api-calls-anatomy/index.md

```diff
@@ -3,39 +3,49 @@ title: API Calls Anatomy
 hide_title: true
 draft: false
 sidebar_label: API Calls anatomy
-sidebar_position: 5
+sidebar_position: 1
 tags:
   - concept
 keywords:
   - trading
   - concept
 description: Deriv API calls Anatomy
 ---
+## Subscribe and Send
 
-## API Call Types
+Every API call has the send functionality and several API calls provide the subscribe functionality.
 
-We provide two types of API calls, `Subscription` and `Send Once` ( Fire and Forget )
+### Subscribe
 
-### Subscription
+Several API calls provide `subscribe` functionality, they create stream of messages for you, which means when you subscribe to them every time that particular event happens for example [Tick History](https://api.deriv.com/api-explorer#ticks_history), you'll get the updated values and data.
 
-Some API calls provide `subscription` ability, which means when you subscribe to them every time that particular event happens for example [Tick History](https://api.deriv.com/api-explorer#ticks_history), these API calls have an `optional` `subscription` field, which if you pass `1` to them, the subscription starts. most of the time data provided by this type of calls will be considered as data source for other calls or features.
+In the message stream from `subscribe` there is a field called `subscription`, this is the `Stream ID`. with this ID you can identity the message stream in your logic and stop the stream with `Forget` and `Forget All` API calls.
 
-### Send Once
+These API calls have an `optional` `subscribe` field, and if you pass `1` to them, the subscription will start and server will continue to send the requested data until you call the `forget`  for your API call you want to `unsubscribe` or `forget`.
 
-`Send Once` API calls are just for getting some data as the name says for once. so if you want to get the updated data you have to send the API call again and get the updated data, most of the time API calls of this type will be used based on other other calls responses or even UI event events such as `Click`, `Scroll`, etc.
+Usually data provided by this type of calls will be considered as data source for other API calls or features.
 
-In order to make it easier for you to handle the `request` and `response` flow of your websocket connection, every deriv websocket API calls has a general structure. you can use them for caching, validation, request and response synchronization, etc.
+### Send
+If you call an API call with `send` functionality, server will only send back the requested data one time. so if you need to get the updated data you have to send the API call again, usually this method will be used based on other other API call responses or UI events such as `Click`, `Scroll`, etc.
 
-## Request Data Anatomy
+### Forget
+If you want to stop the message Stream created by `subscribe` you have to call `Forget` API call with the correct `Stream ID`, or you can call `Forget All` API call to stop streams by their `Method name`.
```

Comment:
Text adjustment: 

If you want to stop the message Stream created by `subscribe`, you will have to call the `Forget` API call with the correct `Stream ID`. Otherwise, you can use the `Forget All` API call to stop streams by their `Method name`.

Reply:
done